README.md

```diff
@@ -25,14 +25,14 @@ On Linux:
 
 1. `git clone` this repo
 2. `docker-compose up`
-3. Go to `http://localhost:4000` to test 
+3. Go to `http://localhost:4000` to test
 
 On OS X, using the [docker-osx-dev](https://github.com/brikis98/docker-osx-dev) project:
 
 1. `git clone` this repo
 2. In one tab: `docker-osx-dev`
 3. In another tab: `docker-compose up`
-4. Go to `http://dockerhost:4000` to test
+4. Go to `http://default:4000` to test
```

Comment:
Ok, that makes total sense because I am indeed using `default` as my docker machine name.  Updated docs in https://github.com/gruntwork-io/gruntwork-io.github.io/commit/6dcb5acde63bac43adb66e3ac59b2cfbbfe70cd8.